src/actions/commands/actions.ts

```diff
@@ -653,7 +653,7 @@ class CommandMoveHalfPageUp extends CommandEditorScroll {
 @RegisterAction
 export class CommandInsertAtCursor extends BaseCommand {
   modes = [ModeName.Normal];
-  keys = ['i'];
+  keys = [['i'], ['<insert>']];
```

Comment:
Oh cool. I don't know why I was surprised at first that the key maps to `<insert>`.